runtimes/rust/src/stubs.rs

```diff
@@ -55,7 +56,7 @@ pub trait Interface<'a> {
     /// Invokes the specified method handler of the service instance.
     fn dispatch(
         &self,
-        cx: CallContext<'a>,
+        cx: ExecutionContext<'a>,
```

Comment:
Nit / off-topic: `cx` can be changed to `ctx` or `context`, since it's what's used throughout the codebase.